gcloud/logging/client.py

```diff
@@ -112,6 +114,7 @@ def list_entries(self, projects=None, filter_=None, order_by=None,
                   more topics can be retrieved with another call (pass that
                   value as ``page_token``).
         """
+        # pylint: disable=too-many-branches
```

Comment:
> Ditto here, why did it become necessary?

At some point, I was suppressing an actual pylint failure here:  it would have been silly to split that function up.  Now, it appears to be unnecessary (maybe merged changes on `master`?)

> Also where is the enable= statement?

[Disabling comments are block-scoped](https://docs.pylint.org/faq.html#is-it-possible-to-locally-disable-a-particular-message).  Moot after 3101fb0.